src/interface/src/app/treatments/map-stands/map-stands.component.ts

```diff
@@ -72,16 +81,22 @@ export class MapStandsComponent implements OnChanges, OnInit {
 
   selectedStands$ = this.selectedStandsState.selectedStands$;
   treatedStands$ = this.treatedStandsState.treatedStands$;
+  sequencedStandsId$ = this.treatedStandsState.sequenceStandsIds$;
```

Comment:
did you mean to call this sequencedStandsIds or sequenceStandIds?

Reply:
hm which one do you think sounds better?

Reply:
Either is okay, but I'd vote for sequenceStands for consistency